checklists/avd_checklist.en.json

```diff
@@ -643,11 +643,20 @@
       "category": "Storage",
       "subcategory": "FSLogix",
       "text": "If FSLogix Cloud Cache is used, move the Cache directory to temporary drive.",
-      "description": "Cloud Cache uses local disk as cache and may generate lot of pressure on the VM disk. Recommended to leverage the power of tempoary (and locally attached) VM disk, if possible based on the VM SKU. ",
+      "description": "Cloud Cache uses local disk as cache and may generate lot of pressure on the VM disk. Recommended to leverage the power of temporary (and locally attached) VM disk, if possible based on the VM SKU. ",
       "guid": "b2d1215a-e114-4ba3-9df5-85ecdcd9bd3b",
       "severity": "Medium",
       "link": "https://docs.microsoft.com/fslogix/cloud-cache-configuration-reference"
     },
+    {
+      "category": "Storage",
+      "subcategory": "FSLogix",
+      "text": "If Azure NetApp Files storage is used, check AD Site name setting in the AD Connection.",
+      "description": "An Active Directory Site should be created for the Azure virtual network environment where ANF subnet will be created, and that site name should be specified in the ANF connection property when executing the join procedure as explained in the reference article.",
+      "guid": "b2d1215a-e114-4ba3-9df5-85ecdcd9bd3b",

```

Comment:
It looks like this GUID is the same as in line 647?

Reply:
Ouch! I used copy&paste the previous element to create a template and then modified. Very bad, now I understand it. How to fix it? Removing only that column value?

Reply:
@erjosito when creating/adding a new element, how to deal with "guid" attribute? We should include with empty "" value or not including it at all? How and when that value is generated?

Reply:
@igorpag The way I do it, I add the new checklist item in the spreadsheet leaving the GUID column blank, and then I export the spreadsheet to JSON (to the avd_checklist.en.json file). That will generate a new GUID. Alternatively you can just generate a new GUID offline (for example in Linux with the command `uuidgen`), and then put this in the JSON.

For this PR I would suggest to generate a new random GUID (for example with `uuidgen`) and commit it to your fork igorpag:main.

Reply:
Done!